content/riak/kv/2.1.4/using/cluster-operations/backing-up.md

```diff
@@ -126,31 +132,31 @@ Bitcask | `/opt/riak/data/bitcask`
 LevelDB | `/opt/riak/data/leveldb`
 Ring | `/opt/riak/ring`
 Configuration | `/opt/riak/etc`
+Cluster Metadata | `/opt/riak/cluster_meta`
+Search | `/opt/riak/yz`
 Strong consistency | `/opt/riak/data/ensembles`
 
-> **Note on strong consistency directories**
->
-> The listings above show directories for data related to Riak's
-[strong consistency][use ref strong consistency] feature. This feature is purely optional, so `/ensembles` directories will not exist in your installation if this feature is not being used. For more information, see [Using Strong Consistency](/riak/kv/2.1.4/developing/app-guide/strong-consistency) and [Managing Strong Consistency](/riak/kv/2.1.4/using/cluster-operations/strong-consistency).
+{{% note title="Note on strong consistency directories" %}}
+The listings above show directories for data related to Riak's [strong consistency][use ref strong consistency] feature. This feature is purely optional, so `/ensembles` directories will not exist in your installation if this feature is not being used. For more information, see [Using Strong Consistency](/riak/kv/2.1.4/developing/app-guide/strong-consistency) and [Managing Strong Consistency](/riak/kv/2.1.4/using/cluster-operations/strong-consistency).
```

Comment:
Strong Consistency isn't optional. It's deprecated. We should remove it, imo, from this page. 

Thoughts, @angrycub @jpease ?